moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -87,18 +94,32 @@ const ProjectsListViewTable = () => {
     storageKey: "mopedProjectListColumnConfig",
   });
 
-  const { columns, columnsToReturnInQuery } = useColumns({ hiddenColumns });
+  const { columns, columnsToReturnInListView, visibleColumnsToReturnInExport } =
```

Comment:
nice - glad to see this worked out, and i think the var name makes it clear to me what is happening here!